aggregator/src/bin/aggregator.rs

```diff
@@ -32,13 +32,28 @@ async fn main() -> Result<()> {
             .response_headers()
             .context("failed to parse response headers")?;
 
+        let mut cfg = ctx.config.aggregator_config();
+
+        if cfg.taskprov_config.enabled && cfg.taskprov_config.global_hpke_keypair.is_none() {
+            match datastore
+                .run_tx_with_name("get_global_hpke_config", |tx| {
+                    Box::pin(async move { tx.get_global_hpke_keypair().await })
+                })
+                .await?
```

Comment:
I opted to just grab the config once at start-up time. Thus we have to restart the process if the key is changed. I figure this is a once-a-year kind of thing so we can be spared cache + cache invalidation logic.